package.json

```diff
@@ -134,6 +135,7 @@
     "@types/webpack-env": "1.14.1",
     "babel-loader": "8.0.6",
     "chai": "4.1.2",
+    "chai-datetime": "1.5.0",
```

Comment:
Do we already have any datetime library in the project? I would avoid external dependencies if possible.

Reply:
It's library for date assertion for tests, so I can write `expect(date).to.equalDate(expected)`